tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -1133,8 +1143,50 @@ def build(self, input_fn):
       raise RuntimeError("input_fn is None. Method build() needs input_fn "
                          "to be specified in order to build TensorRT engines")
 
+    def _rebuild_func():
+      # Rebuild function from graph_def.
+      reset_converted_func = wrap_function.function_from_graph_def(
+          self._converted_graph_def,
+          [tensor.name for tensor in self._converted_func.inputs],
+          [tensor.name for tensor in self._converted_func.outputs])
+      reset_converted_func.graph.structured_outputs = nest.pack_sequence_as(
+          self._converted_func.graph.structured_outputs,
+          reset_converted_func.graph.structured_outputs)
```

Comment:
This code looks the same as the end of "convert" function to me. Can we make this function return reset_converted_func instead so that we can reuse it for "convert"? Need to rename the variable in this case.